pyecharts/charts/composite_charts/page.py

```diff
@@ -65,6 +99,39 @@ def _prepare_render(self):
             if c.theme not in ThemeType.BUILTIN_THEMES:
                 self.js_dependencies.add(c.theme)
 
+        charts_id = []
+        if self.layout == _MARK_FREEDOM_LAYOUT:
+            self.download_button = True
+            for c in self:
+                charts_id.append("'{}'".format(c.chart_id))
+                self.add_js_funcs(
+                    # make charts resizable and draggable
+                    '$("#%s")'
```

Comment:
charts maybe resizeable and draggable but code is not readable.. triple quotes?

Reply:
jinja2 can also do templating function

Reply:
using python3 f-format now.